frontend/src/components/Itinerary/ItineraryWrapper/ItineraryWrapper.jsx

```diff
@@ -11,5 +11,5 @@ export default function ItineraryWrapper() {
   const { response, loading } = useGet(URL);
 
   // show spinner while data is being fetched
-  return <div>{loading || !response ? <Spinner /> : <Itinerary itineraryData={response.data} />}</div>;
+  return <>{loading || !response ? <Spinner /> : <Itinerary itineraryData={response.data} />}</>;
```

Comment:
currently the React file structure here breaks convention with the ItineraryWrapper folder being inside the Itinerary folder, even though the Wrapper is the outer component. Can this be refactored?